examples/armv4t/gdb/mod.rs

```diff
@@ -482,6 +495,76 @@ mod custom_arch {
             Some("never gets returned")
         }
 
+        // (LLDB extension) TODO
+        fn register_info<'a>(reg_id: usize, write_info: Callback<'a>) -> Option<CallbackToken<'a>> {
+            match ArmCoreRegId::from_raw_id(reg_id) {
```

Comment:
this should be using `ArmCoreRegIdCustom`, and include `Custom` register (but not the `Time` register). See the implementation of target.xml override for Armv4tCustom for details